### ISSUE TITLE
First-person perspective, right-hand dataset and model for recognizing static hand poses.

### ISSUE THREAD

**Issue by dev-td7** [dev-td7/Indian-Sign-Language-Translator#1]
Currently, the pre-trained model in this repository is capable of recognizing hand poses from a second-person perspective. Meaning, an image taken by you of some other person making a hand pose is the ideal image for recognition. 

Need to create a **new model and dataset of images** of the same hand poses taken from a first-person perspective, i.e. images taken by your camera held by your left hand of your right hand. Refer to the [reference image](https://github.com/dev-td7/Indian-Sign-Language-Translator/blob/master/Indian%20Sign%20Language%20Hand%20Poses.png?raw=true) provided in the repository for Indian Sign Language Hand Poses.

You need to create a dataset first. 300 images per label is the ideal number per image. Use the training scripts in the `train/` folder to create a new model.

**What are hand poses?**
Static signs made by your hand - imagine you representing the number '5' from your hand is called a hand pose.
